pandas/tests/frame/methods/test_astype.py

```diff
@@ -136,6 +137,15 @@ def test_astype_cast_nan_inf_int(self, val, dtype):
         with pytest.raises(ValueError, match=msg):
             df.astype(dtype)
 
+    @pytest.mark.parametrize("dtype", ["Int64", "Int32", "Int16"])
+    @pytest.mark.parametrize("val", [np.nan, NA])
```

Comment:
use `nulls_fixture` instead and `any_nullable_int_dtype` fixture

Reply:
Hey, here in `nulls_fixture`, the test case fails for `pd.NaT` because it is inferred as `datetime64`. I think it is expected behavior but would like your opinion on it.

code snippet for reference
```python
>>> df = pd.DataFrame({"a": ["foo"], "b": [pd.NaT]})
>>> df.dtypes
a            object
b    datetime64[ns]
dtype: object
>>> df.loc[:, "b"] = df.loc[:, "b"].astype('Int64')
TypeError: datetime64[ns] cannot be converted to an IntegerDtype
```

Reply:
@arw2019  @jreback , Any updates on this one? would appreciate some help and direction.

Reply:
Could use `pytest.skipif` for the datetime case
or test for the `TypeError` somehow but I'm not sure that's necessary